packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Checkbox.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+    * The checked property of the Input element. Use this to generate a controlled Checkbox Element.
+    */
+  checked: PropTypes.bool,
+  /**
+    * The checked property of the Input element. Use this to generate an uncontrolled Checkbox Element.
+    */
+  defaultChecked: PropTypes.bool,
+  /**
+    * The id of the input field.
+    */
+  id: PropTypes.string,
+  /**
+    * Additional attributes for the input object.
+    */
+  inputAttrs: PropTypes.object,
+  /**
+   * Whether the checkbox element is disabled or not.
+   */
+  isDisabled: PropTypes.bool,
+  /**
+    * Whether the checkbox element is inline or not.
+    */
+  isInline: PropTypes.bool,
+  /**
+    * Whether the label is hidden or not.
+    */
+  isLabelHidden: PropTypes.bool,
+  /**
+    * Text of the label.
+    */
+  labelText: PropTypes.node.isRequired,
+  /**
+    * Additional objects for the text object.
+    */
+  labelTextAttrs: PropTypes.object,
```

Comment:
If consumers want to pass in additional attributes for the label should they pass in a node? ( Do we need this prop or should we remove it?) 

<Checkbox label={<span style={{ color: blue }} />

If we do keep this prop, and I may have missed it somewhere, it looks like we are only honoring the className passed in through labelTextAttrs, any additional key values pairs are discarded.

Reply:
Good catch, I forgot to pass/spread `labelTextAttrs` into the span. Updated